archetypes/author.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "" # Name
+image: ""
+email: ""
+social:
+  - icon : "ti-linkedin" # themify icon pack : https://themify.me/themify-icons
+    link : "#" # Linkedin
+  - icon : "ti-twitter-alt" # themify icon pack : https://themify.me/themify-icons
+    link : "#" # Twitter
+  - icon : "ti-github" # themify icon pack : https://themify.me/themify-icons
+    link : "#" # Github
```

Comment:
Could we add the link prefixes so that the user simply has to type in their username? E.g. `"https://www.linkedin.com/in/"` for LinkedIn instead of `"#"` and `"https://github/com/"` for GitHub, etc.